audit/auditLog.json

```
@@ -58,11 +65,23 @@
     }
   },
   "auditedContracts": {
+    "EmergencyPauseFacet": {
```

Biome (1.9.4)

[error] The key EmergencyPauseFacet was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)